packages/wrangler/src/update-check.ts

```diff
@@ -0,0 +1,22 @@
+import chalk from "chalk";
+import checkForUpdate from "update-check";
+import pkg from "../package.json";
+
+export async function updateCheck(): Promise<JSX.Element | null> {
+  let update = null;
+  try {
+    // default cache for update check is 1 day
+    update = await checkForUpdate(pkg, {
+      distTag: "beta",
+    });
+  } catch (err) {
+    console.error(`Failed to check for updates: ${err}`);
+  }
+
+  if (update)
```

Comment:
Does this mean we'll always show this message if there's an update? Do you suppose we only want to show this message once a day?

Reply:
The default is it checks once a day.